.github/workflows/tests.yml

```diff
@@ -208,7 +208,7 @@ jobs:
             fi
           done
           if [[ $cs_fix_files ]]; then
-            bin/php-cs-fixer fix --config=.php-cs-fixer.php -v --dry-run --using-cache=no --show-progress=dots --diff $cs_fix_files
+            bin/php-cs-fixer fix --config=.php-cs-fixer.php --dry-run --using-cache=no --show-progress=dots --diff $cs_fix_files
```

Comment:
Why is the verbose flag removed here?

Reply:
Because it's an unnecessary info for the build process. The progress of the whole build is visible in the github interface.